support-frontend/assets/components/svgs/dropdownArrowUp.jsx

```diff
@@ -0,0 +1,11 @@
+// @flow
+
+import React from 'react';
+
+export default function SvgDropdownArrowUp() {
```

Comment:
You can actually use .svg files now, this is a legacy way of doing svgs. See https://github.com/guardian/support-frontend/blob/d6ea91426b0503eccaea3b928db2fbb1df7aa871/support-frontend/assets/components/consentBanner/consentBanner.jsx#L13

Reply:
Thanks Rupert, that's good to know - I will fix it. 😄